### ISSUE TITLE
conf-libssl: failed: ssl_stubs.c: 'TLS1_3_VERSION' undeclared

### ISSUE THREAD

**Issue by jrochel** [ocaml/opam-repository#18604]
On Alpine libressl is used. This seems to cause problems:

https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/f1c806126ee8b8081aaeb26786069169a7f429d5

**Comment by zoggy** [comment]
Same here:
````
$ uname -a
Linux raspberrypi 4.19.66-v7+ #1253 SMP Thu Aug 15 11:49:46 BST 2019 armv7l GNU/Linux
````
with libss(-devl) Version: 1.1.0l-1~deb9u3

**Comment by kit-ty-kate** [comment]
This was fixed by #18964
